cwf/cloud/helm/cwf-module/templates/analytics.service.yaml

```diff
@@ -0,0 +1,31 @@
+{{/*
+# Copyright 2020 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+*/}}
+
+{{- include "modulelib.service" (list . "analytics.service") -}}
+{{- define "analytics.service" -}}
+metadata:
+  name: {{ .Release.Name }}-analytics
+  labels:
+    {{- with .Values.analytics.service.labels }}
+{{ toYaml . | indent 4}}
+    {{- end}}
+  {{- with .Values.analytics.service.annotations }}
+  annotations:
+{{ toYaml . | indent 4}}
+  {{- end }}
+spec:
+  ports:
+    - name: grpc
+      port: 9200
```

Comment:
I think even now we should frontend on a common port for grpc and obsidian traffic. Until we split up the supercontainer we'll have to backend on the right corresponding port still